cnf/gac.in

```diff
@@ -114,7 +114,14 @@ c_link_dyn () {
     output_la=${1%.so}.la
     output_dir=$(dirname $1)
     ${c_dyn_linker} ${GAP_LDFLAGS} -o "$output_la" $2 ${c_addlibs} || exit 1
-    cp -r $output_dir/.libs/*.so $output_dir
+    if [ X"$SYS_IS_CYGWIN32" = X"yes" ] ; then
+        # GAP assumes shared libraries end in .so
+        for dllfile in `cd $output_dir/.libs; ls *.dll`; do
+            cp -r $output_dir/.libs/$dllfile $output_dir/${dllfile%.dll}.so
+        done
+    else
+        cp -r $output_dir/.libs/*.so $output_dir
```

Comment:
I'd drop the `-r` -- yes, the original code used it, but I'd also drop it there: It is a pre-POSIX way of suggesting a recursive copy, but that's not what we want here (and it's non-POSIX...)